src/power_grid_model_io/functions/phase_to_phase.py

```diff
@@ -40,30 +40,34 @@ def power_wind_speed(  # pylint: disable=too-many-arguments
     nominal_wind_speed: float = 14.0,
     cutting_out_wind_speed: float = 25.0,
     cut_out_wind_speed: float = 30.0,
+    axis_height: float = 30.0,
 ) -> float:
     """
     Estimate p_ref based on p_nom and wind_speed.
 
     See section "Wind turbine" in https://phasetophase.nl/pdf/VisionEN.pdf
     """
 
+    # Calculate wind speed at the axis height
+    wind_speed_height = wind_speed * (axis_height / 10) ** 0.143
```

Comment:
Minor, but can we call it `wind_speed_at_axis_height`?

Reply:
Or just `wind_speed *= (axis_height / 10) ** 0.143`.

Reply:
Updated as per 2nd comment